app/components/NavTree.js

```diff
@@ -24,18 +24,18 @@ export default class SchoolsNavTree extends React.Component {
   }
 
 
-  renderSubTree(node, boundaryHierarchy, visitedBoundaries, depth) {    
-    const boundaryDetails = this.props.boundaryDetails;        
+  renderSubTree(node, boundaryHierarchy, visitedBoundaries, depth) {
+    const boundaryDetails = this.props.boundaryDetails;
     if (boundaryDetails[node].depth == depth && depth < 5) {
       if (node && $.inArray(node, visitedBoundaries) < 0) {
         var children = boundaryHierarchy[node];
         visitedBoundaries.push(node);
 
         var boundary = this.props.boundaryDetails[node];
-        const label = <Link key={ boundary.name || boundary.id } to={ boundary.path } onClick={ this.props.onBoundaryClick.bind(null, boundary) }><span className="node"> { boundary.label || boundary.name || boundary.first_name} </span></Link>;
+        const label = <Link key={ boundary.name || boundary.id } to={ boundary.path }><span className="node"> { boundary.name || boundary.first_name} </span></Link>;
```

Comment:
Actually, it is not even showing up here..I'd added {boundary.label || boundary.name || boundary.id} in the label..